test/ssmParameter.go

```diff
@@ -0,0 +1,30 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build integration
+// +build integration
+
+package test
+
+import (
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ssm"
```

Comment:
Would it be better to use SSM v2 ?

Reply:
yeaah Andrew had a similar comment. I will have to go over every non v2 stuff [TODO]